dependencies.yaml

```diff
@@ -17,36 +17,40 @@ dependencies:
       lang: scala
 
   com.fasterxml.jackson.core:
-    jackson:
+    jackson-databind:
       lang: java
-      modules: [ "core", "databind" ]
+      version: "2.10.5.1"
+
+    jackson-core:
+      lang: java
+      version: "2.10.1"
```

Comment:
this is the latest version of jackson that circe for 2.11 supports.

We need to move to 2.12 or 2.13 but that is not in scope on this PR.